packages/integrations/node/readme.md

```diff
@@ -0,0 +1,25 @@
+# @astrojs/node
```

Comment:
There are probably more docs to add here, but okay to tackle later. One thing that jumps out is error handling: Should I ever expect `next` to be called, and if so, what would it be called with? Are 404s handled by the adapter?

Reply:
This is using the Express convention but is not tied to express. This is similar to Remix's adapter. No, it doesn't handle 404s, `next()` will be called in that case. If you were using Express you'd create middleware to handle other routes and then defer to a 404 route at the end. `next` is also called in case of an error.

Reply:
fwiw, I would expect Astro to handle 404s with the Astro 404 page concept.